packages/meteor-accounts-saml/saml_server.js

```diff
@@ -170,17 +170,21 @@ Accounts.registerLoginHandler(function(loginRequest) {
 	}
 });
 
-Accounts.saml._loginResultForCredentialToken = {};
-
 Accounts.saml.hasCredential = function(credentialToken) {
-	return _.has(Accounts.saml._loginResultForCredentialToken, credentialToken);
+	const data = RocketChat.models.CredentialTokens.findOneById(credentialToken);
+	return Boolean(data);
```

Comment:
Forgive my momentary ignorance.  :) findOneById will return null or undefined here right?  No chance of {} or any other result?